pyo3_special_method_derive_macro/src/lib.rs

```diff
@@ -369,12 +372,13 @@ pub fn str_derive(input_stream: TokenStream) -> TokenStream {
 ///     hash: u32,
 /// }
 /// ```
-#[proc_macro_derive(AutoDisplay, attributes(pyo3_smd, pyo3_smd_str, format))]
+#[proc_macro_derive(AutoDisplay, attributes(skip, pyo3_fmt_no_skip, format))]
```

Comment:
this is making me think we could have the equivalent `no_skip`  instead of pyo3_fmt_no_skip. Will put this as a todo for me!

Reply:
Sounds good!